### ISSUE TITLE
Whole dayItem should not be clickable because it's counter-intuitive

### ISSUE THREAD

**Issue by ghostbuster91** [elpassion/el-peon-android#19]

**Comment by jozzi05** [comment]
What do you suggest? For me it works great, but it has been implemented by me, so maybe it can be clear only for me. I don't know. Maybe there is a way to ask UX designer for a help?

**Comment by ghostbuster91** [comment]
Agree, the best way to solve this problem would be to ask a designer.
In the mean time I think that this is breaking some UX rules, because what's the purpose of having this plus icon there? (I know, you'll say that's an indicator, but for me it looks like something clickable). 

The AB test would be probably also good.
